poms/bom-deployment/pom.xml

```diff
@@ -822,6 +823,16 @@
                 <artifactId>camel-quarkus-telegram-deployment</artifactId>
                 <version>${camel-quarkus.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.camel.quarkus</groupId>
+                <artifactId>camel-quarkus-tika-deployment</artifactId>
+                <version>${camel-quarkus.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-tika-deployment</artifactId>
+                <version>${quarkus.version}</version>
+            </dependency>
```

Comment:
Plz move this one to the top where the mysql driver is and add a link to the quarkus PR where it is fixed in their BOM.